src/cpp/README.md

```diff
@@ -30,11 +30,23 @@ To add gRPC as a dependency in bazel:
   grpc_deps()
   ```
 
-NOTE: currently bazel is only supported for building gRPC on Linux.
+## cmake
+
+`cmake` is your best option if you cannot use bazel. It supports building on Linux, MacOS and Windows (official support) but also has a good chance of working on other platforms (no promises!). `cmake` has good
+support for crosscompiling and can be used for targeting Android platform.
+
+If your project is using cmake, there are several ways to add gRPC dependency.
```

Comment:
I'd like to split this section up into "depending on gRPC" (find_package, always) and building gRPC as a dependency (superbuild, git submodule, etc). But IMO that change can wait for a subsequent PR.

Reply:
Let's propose your improvements in a followup PR (after we discuss the outline). This PR is only trying to fix inaccuracies and reorganizing things I knew were wrong.